conf/machine/x86-generic-64.conf

```diff
@@ -6,7 +6,7 @@ DEFAULTTUNE ?= "corei7-64"
 require conf/machine/include/tune-corei7.inc
 require conf/machine/include/qemu.inc
 require conf/machine/include/qemuboot-x86.inc
-QB_OPT_APPEND = "-vga vmware -show-cursor -usb -usbdevice tablet -device virtio-rng-pci -smp 2"
+QB_OPT_APPEND = "-vga vmware -usb -usbdevice tablet -device virtio-rng-pci -smp 2"
```

Comment:
We should pull this inline with the append in qemuboot-x86.inc which just has the -usb -device usb-tablet setting, but keep the -smp 2.

Reply:
added changes for x86-atom conf